src/cow-react/pages/Swap/SwapMod.tsx

```diff
@@ -402,6 +403,15 @@ export default function Swap({ history, location, className }: RouteComponentPro
               <SwapButtons {...swapButtonsContext} />
               {isExpertMode ? <ErrorMessage error={swapErrorMessage} /> : null}
             </BottomGrouping>
+            {!swapErrorMessage && nativeInput && (
```

Comment:
@W3stside we also have `swap/NewSwapWidget/index.html` and the banner suppose to be there too.
I propose:
1. Add a new state to `SwapButtonState`: `SwapButtonState.HasEnoughWrappedForEthFlow` and detect it in `getSwapButtonState()`
2. Render `EthFlowBanner` in the`SwapButtons` component

Reply:
👍

Reply:
good point!